src/components/Button.tsx

```diff
@@ -15,21 +17,18 @@ const Button: React.FC<ButtonProps> = ({text, onPress, color, textColor}) => (
   </ButtonWrapper>
 )
 
-interface IButtonWrapper {
-  color?: string
-}
-const ButtonWrapper = styled.TouchableOpacity<IButtonWrapper>(({theme, color}) => ({
+// interface IButtonWrapper {
+//   color?: string
+// }
+const ButtonWrapper = styled(TouchableOpacity)<{theme: ITheme; color?: string}>(({theme, color}) => ({
```

Comment:
you don't need to define the type of theme. Because I did config default theme for styled